docs/user/_sidebar.md

```diff
@@ -1,5 +1,5 @@
-- [Keda module](/docs/user/README.md)
-- [KEDA configuration](/docs/user/01-20-configuration.md)
-- [Keda module footprint](/docs/user/04-10-footprint.md)
-- [KEDA demo application](/docs/user/04-20-demo-applications.md)
-- [Keda CR Conditions](/docs/user/05-01-conditions.md)
+- [Keda module](README.md)
```

Comment:
```suggestion
- [Home](/)
- [Keda module](README.md)
```

Reply:
Adding Home link will allow to get back to the main website when user enters Keda docs.
I also fixed a link to demo application (4-20-demo... -> 04-20-demo...)
You can test your branch with docsify to spot the problems:
- install docsify: `npm i docsify-cli -g`
- checkout kyma-project/kyma repository (git clone)
- start website locally from kyma root folder: `docsify serve docs`
- modify alias in the docs/index.html to point to your branch:
    ```
    '/keda-manager/(.*)': 'https://raw.githubusercontent.com/pbochynski/keda-manager/docsify/docs/$1'
    ```
    to
    ```
    '/keda-manager/(.*)': 'https://raw.githubusercontent.com/grego952/keda-manager/relative-links/docs/$1'
    ```

Reply:
Ok, thanks.

Reply:
Looks like everything works fine. 
Thank you @pbochynski for your support!